### ISSUE TITLE
performing Joins

### ISSUE THREAD

**Issue by illusion1912** [vertica/vertica.dplyr#15]
Is there a way to create a join between a local dataframe and a dataframe created from a vertica table, without fetching the vertica dataframe to local? 
Note: I dont have write access to the Vertica databse

**Comment by vishrutg** [comment]
No, there isn't any way without moving either the local data frame to vertica or fetching the table from vertica
